deploy/kubernetes/tracee.yaml

```
@@ -33,6 +33,18 @@ spec:
         - name: lib-modules
           mountPath: /lib/modules/
           readOnly: true
+        resources:
+          limits:
+            cpu: 500m
+            memory: 300Mi
+          requests:
+            cpu: 350m
+            memory: 50Mi
+      tolerations:
+        - effect: NoSchedule
+          operator: Exists
+        - operator: "Exists"
+          effect: "NoSchedule"
```

Is there any difference between the two tolerations?? I guess you wanted to use NoExecute effect:

```suggestion
          effect: "NoExecute"
```

Also consider tolerating any taints with:

```yaml
tolerations:
  - operator: Exists
```

NIT: Use / don't use double quotes consistently 🤓